ox-pandoc.el

```diff
@@ -1512,6 +1515,30 @@ version. If nil, no checks are performed and no warnings generated."
   "Export as zimwiki."
   (interactive) (org-pandoc-export 'zimwiki a s v b e t))
 
+(defcustom org-pandoc-options-for-twiki nil
```

Comment:
This code repetition is getting out of control, honestly. It would be nice to write a macro that would define all that boilerplace automatically, based on the export format name.